[2014-03-31 07:31:29] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:26:48] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:42:06] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:47:29] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:49:52] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:49:54] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:50:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:50:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:50:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:50:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:50:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:50:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:50:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:50:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:50:08] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 08:52:57] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:01:51] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:01:55] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:03:29] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:03:32] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:06:31] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:78 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(78): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 78, Array) #1 /var/www/arp_dev/app/controllers/admin/MapaymentController.php(104): Controllers\Admin\MapaymentController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MapaymentController->getPay('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getPay', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:12:20] log.ERROR: exception 'ErrorException' with message 'Undefined variable: conversionrate' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:74 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(74): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 74, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-31 09:12:43] log.ERROR: exception 'ErrorException' with message 'Undefined variable: conversionrate' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:91 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(91): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 91, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-31 09:29:40] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::getConversionrate()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('getConversionra...', Array) #1 [internal function]: Illuminate\Database\Query\Builder->getConversionrate('1021') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('getConversionra...', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->getConversionrate('1021') #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2653): call_user_func_array(Array, Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(130): Illuminate\Database\Eloquent\Model->__call('getConversionra...', Array) #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(130): Mainvoice->getConversionrate('1021') #8 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #19 {main} [] []
[2014-03-31 09:29:58] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::getConversionrate()' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1658 Stack trace: #0 [internal function]: Illuminate\Database\Query\Builder->__call('getConversionra...', Array) #1 [internal function]: Illuminate\Database\Query\Builder->getConversionrate('1021') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(786): call_user_func_array(Array, Array) #3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('getConversionra...', Array) #4 [internal function]: Illuminate\Database\Eloquent\Builder->getConversionrate('1021') #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(2653): call_user_func_array(Array, Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(130): Illuminate\Database\Eloquent\Model->__call('getConversionra...', Array) #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(130): Mainvoice->getConversionrate('1021') #8 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #19 {main} [] []
[2014-03-31 09:30:58] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:33:14] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:33:45] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1020') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1020') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1020') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:33:59] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1023') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1023') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1023') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:37:23] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1023') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1023') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1023') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:39:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1016') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1016') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1016') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:39:18] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1009') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1009') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1009') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:39:27] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1005') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1005') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1005') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:39:36] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1016') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1016') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1016') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:39:43] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:42:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:47:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:47:21] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:47:25] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:48:21] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '.'' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-31 09:48:30] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:11] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:14] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:14] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:14] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:14] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:14] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:51:14] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:52:07] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 09:53:17] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:93 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(93): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 93, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getConversionrate('1021') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1021') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1021') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 10:18:55] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1019') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1019') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1019') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 10:21:51] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:97 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(97): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 97, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Controllers\Admin\MainvoiceController->getConversionrate('1019') #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1019') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1019') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-31 11:12:36] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare Controllers\Admin\MapaymentController::getPay()' in /var/www/arp_dev/app/controllers/admin/MapaymentController.php:117 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-31 11:13:04] log.ERROR: exception 'ErrorException' with message 'Undefined variable: conversionrate' in /var/www/arp_dev/app/storage/views/b0458d0525ef97476c6205ceb5313000:152 Stack trace: #0 /var/www/arp_dev/app/storage/views/b0458d0525ef97476c6205ceb5313000(152): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 152, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getPay', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getPay', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1024') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-31 15:29:59] log.ERROR: exception 'ErrorException' with message 'Undefined variable: id' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:129 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 129, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getShow('1024') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1024') #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-31 15:30:18] log.ERROR: exception 'ErrorException' with message 'Undefined variable: id' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:129 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(129): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 129, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getShow('1024') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1024') #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-31 15:30:49] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Controllers\Admin\Mapayment' not found' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:131 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-31 15:52:13] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$'' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:643 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-31 15:58:41] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$invoice' (T_VARIABLE), expecting ',' or ';'' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:642 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-31 15:59:05] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'To' (T_STRING), expecting ',' or ';'' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:643 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
